facebook-commerce.php

```diff
@@ -927,6 +935,26 @@ public function delete_fb_product( $product ) {
 		delete_post_meta( $product_id, self::FB_PRODUCT_GROUP_ID );
 	}
 
+	/**
+	 * Deletes a product from Facebook when status is changed to draft.
+	 *
+	 * @param string   $new_status
+	 * @param string   $old_status
+	 * @param \WP_post $post
+	 */
+	public function delete_draft_product( $new_status, $old_status, $post ) {
+
+		if ( ! $post ) {
+			return;
+		}
+
+		if ( $old_status === 'publish' && $new_status === 'draft' ) {
+			$product_id = $post->ID;
+			$this->on_product_delete ( $product_id );
```

Comment:
```suggestion
			$this->on_product_delete ( $post->ID );
```

I think the variable is not needed. but that's my personal preference, you don't have to apply the change :-)

Reply:
I think that this would work with just `$post`.

Because the first line of `on_product_detele` is:

```PHP
	$product = wc_get_product( $product_id );
``` 

and the first param to `wc_get_product` is described as:
```PHP
 * @param mixed $the_product Post object or post ID of the product.
```

Reply:
@budzanowski Are you suggesting we make changes to `on_product_detele` method? If not then we'd get an error since the method accepts int parameter.

Reply:
We could but lets keep it as is and just use what @rawdreeg has suggested.